core/templates/pages/privacy-page/privacy-page.component.html

```diff
@@ -8,7 +8,7 @@ <h1>Privacy Policy</h1>
       mission, Oppia (also "we" or "us") collects some information from its
       users to deliver an interactive and engaging experience. This Privacy
       Policy describes the types of information we collect from users of the
-      Oppia website (<a href="/">https://www.oppia.org</a>)
+      Oppia website (<a href="/" style="text-decoration: underline;">https://www.oppia.org</a>)
```

Comment:
Please don't use inline styles like this, prefer using a CSS class for this page instead. Ditto elsewhere.

Reply:
Just moved all inline styles to CSS class. Thanks for the feedback.

Reply:
@gutoslv Hm, I think there are still inline styles here. Please check the "Files Changed" tab before requesting a review to ensure that what you're showing reviewers is what you intend to commit -- thanks!

Reply:
@seanlip my bad. Linters were failing silently on my local, just fixed it.